Source/Core/VideoCommon/VertexShaderManager.cpp

```diff
@@ -30,27 +30,6 @@
 #include "VideoCommon/VideoConfig.h"
 #include "VideoCommon/XFMemory.h"
 
-alignas(16) static std::array<float, 16> g_fProjectionMatrix;
-
-// track changes
-static std::array<bool, 2> bTexMatricesChanged;
```

Comment:
Can remove `<array>` from cpp since it is now in the header

Reply:
Since `VertexShaderManager.cpp` also uses an `std::array` independent of its class (see eg. the `pos_matrix_ptrs` in `SetConstants()`) that would end up relying on an indirect include from the header, so if for whatever reason you factored out the `#include <array>` from `VertexShaderManager.h` you would suddenly end up with a missing definition for `std::array` in the `.cpp` file. Obviously this is a personal preference to some degree, but I like to (try to, at least) ensure that a `.cpp` file doesn't rely on indirect includes like that.